code/modules/religion/festival/musician_rites.dm

```diff
@@ -0,0 +1,101 @@
+///prototype for rites that tune a song.
+/datum/religion_rites/song_tuner
+	name = "Tune Song"
+	desc = "this is a base."
+	ritual_length = 10 SECONDS
+	favor_cost = 10
+	///if repeats count as continuations instead of a song's end, TRUE
+	var/repeats_okay = TRUE
+	///visible message sent to indicate a song will have special properties
+	var/visible_message
+
+/datum/religion_rites/song_tuner/invoke_effect(mob/living/user, obj/structure/altar_of_gods/AOG)
+	user.AddComponent(/datum/component/smooth_tunes, src, repeats_okay)
+
+/**
+  * Perform the song effect.
+  *
+  * Arguments:
+  * * arg1 - Atom (parent) of the smooth_tunes component. This is limited to the compatible items of said component, which currently includes mobs and objects so we'll have to type appropriately.
+  */
+/datum/religion_rites/song_tuner/proc/song_effect(atom/A, datum/song/S)
+	return
+
+/datum/religion_rites/song_tuner/evangelism
+	name = "Evangelical Hymn"
+	desc = "Spreads the word of your god, gaining favor for each non-holy listener."
+	favor_cost = 0
+
+/datum/religion_rites/song_tuner/evangelism/song_effect(atom/A, datum/song/S)
+	if(!S || !GLOB.religious_sect)
+		return
+	for(var/i in S.hearing_mobs)
```

Comment:
time to perform for a shitton of monkeys

Reply:
i may fix this (you can see by the single letter vars that i haven't reviewed this file)

that is really funny but not exactly promoting "interaction"

Reply:
you could reduce the point gain rate for clientless mobs without entirely eliminating it

sort of like how that one nanite program that generated tech points used to work

Reply:
Could have monkeys fling shit at the performer, because they're too uncultured to appreciate good art

Reply:
i think i'm happy with where it's at now, from my in game testing you'll be able to get a decent amount of points